renderdoc/driver/vulkan/vk_shaderdebug.cpp

```diff
@@ -4245,6 +4318,7 @@ ShaderDebugTrace *VulkanReplay::DebugPixel(uint32_t eventId, uint32_t x, uint32_
   builtins[ShaderBuiltin::DrawIndex] = ShaderVariable(rdcstr(), action->drawIndex, 0U, 0U, 0U);
   builtins[ShaderBuiltin::Position] =
       ShaderVariable(rdcstr(), float(x) + 0.5f, float(y) + 0.5f, 0.0f, 0.0f);
+  builtins[ShaderBuiltin::ViewportIndex] = ShaderVariable(rdcstr(), view, 0U, 0U, 0U);
```

Comment:
This is my prior mistake, not a problem in this PR, but I realise now that `ViewIndex` shouldn't be mapped to the same builtin as `ViewportIndex`. They're two different builtins with two different meanings.

There should be a new `ShaderBuiltin` enum value called I think to be easier to identify `ShaderBuiltin::MultiViewIndex` which should be used in the places you're using it here.

Also - I think this builtin should only be set if we have a view we're looking for, not if its `~0U`. In most cases it should get overwritten by the read value anyway below when we fetch PS inputs, but we should be safe.